src/eth-signer/src/ic_sign.rs

```diff
@@ -166,15 +170,15 @@ impl IcSigner {
 
         let r = U256::from_be_slice(&signature_data[0..32]);
         let s = U256::from_be_slice(&signature_data[32..64]);
-        let v = recovery_id.is_y_odd() as u64 + 27;
+        let y_parity = recovery_id.is_y_odd();
```

Comment:
y_parity = true. Could be misleading. Parity is even or odd

Reply:
done